### ISSUE TITLE
Create Documentation / Wiki

### ISSUE THREAD

**Issue by psifidotos** [psifidotos/workflow-project#69]
It would be very productive to have documentation. I have create in google docs a document:

https://docs.google.com/document/d/1PTGs_kmwDe3Ov331tBgQBDTgIVTB6OMIpvwMzygOhEc/edit#

in order to create a collaborative effort. If anyone is interested just tell...
A team of 2-3 persons is more than enough, I dont think it's goind to be very big...

**Comment by bmihaila** [comment]
Hi

I would want to help with the docu but why don't we take the github wiki here on this site https://github.com/psifidotos/workflow-project/wiki ? You could then link from the main project site to the wiki as the documentation. Somehow the google docs is more like a static document as the documentation should be something that is updated often.

**Comment by psifidotos** [comment]
Yes of course no problem, github says that is like a simple repo. I like
wiki style also.

BTW, try the latest version, it supports global hotkeys, you asked for :-)
...
Óôéò 18 Öåâ 2013 2:11 ð.ì., ï ÷ñÞóôçò "Bogdan Mihaila" <
notifications@github.com> Ýãñáøå:

**Comment by bmihaila** [comment]
Just tried the latest version, really cool! All the things I was missing from activities came now true, thx to you.

Ok, I will then gradually start adding some FAQs to the wiki the next days ... maybe even some stubs for the features. I guess the introduction and Concept parts are more up to you ;)

**Comment by psifidotos** [comment]
Yes of course, dont worry...., once there is some work in the wiki I 
will help....

> I guess the introduction and Concept parts are more up to you ;)

**Comment by bmihaila** [comment]
Hi
I started some documentation https://github.com/psifidotos/workflow-project/wiki/Workflow-Project
At the moment it lists the major features and some FAQ. Needs some polishing I guess and some more FAQ but its a start.

**Comment by bmihaila** [comment]
We will have to think how to organize the wiki. Here some issues:
- subpages for the topics or one big page?
- what things to add there? (installation, links to videos, screenshots etc..)
- merge things from features and interface?
- show screenshots (especially for the interface)? The problem with screenshots is that they need to be kept up to date when the interface changes!
- be concise and short like I wrote things now or with longer descriptions (more like a manual)

**Comment by bmihaila** [comment]
Ooh, one more thing. In the "Concept" paragraph you need to stress the differences and commonalities between (virtual) desktops and Workareas. That there are two different terms and what they mean and that they are sometimes used interchanged was quite confusing for me at the beginning. So you should name them and state what they are and what not and from then on stick with Workareas as to not confuse people.

**Comment by psifidotos** [comment]
> Ooh, one more thing. In the "Concept" paragraph you need to stress the 
> differences and commonalities between (virtual) desktops and 
> Workareas. That there are two different terms and what they mean and 
> that they are sometimes used interchanged was quite confusing for me 
> at the beginning. So you should name them and state what they are and 
> what not and from then on stick with Workareas as to not confuse people.
> 
> Good Idea I will use references from the original text from which this 
> project derived....

**Comment by psifidotos** [comment]
> Hi
> I started some documentation 
> https://github.com/psifidotos/workflow-project/wiki/Workflow-Project
> At the moment it lists the major features and some FAQ. Needs some 
> polishing I guess and some more FAQ but its a start.
> 
> Very good, I will add soon the concept text....

**Comment by psifidotos** [comment]
> We will have to think how to organize the wiki. Here some issues:
> - subpages for the topics or one big page?
> 
> I think that we can stick with one page for the time....
> - what things to add there? (installation, links to videos,
>   screenshots etc..)
> 
> Installation: agree
> Videos & Screenshots: I dont think is necessary we can create links to 
> the original
> project page... 
> http://opentoolsandspace.org/en/projects/development/plasmoid-workflow
> I upload screenshots and videos for each major version there...
> - merge things from features and interface?
> 
> Agree, even though I can not think what to add in the interface...
> - be concise and short like I wrote things now or with longer
>   descriptions (more like a manual)
> 
> I prefer short texts, in order to be read easily....

**Comment by psifidotos** [comment]
Ok... The concept is ready take a look at it....

One more thing, I changed my opinion (Concept is very big and I can not make it smaller... :) ), I believe we could separate the wiki in the following:
Main page with Introduction and links for all the rest sub-sections e.g.:
Concept,
Features,
FAQ

what do you think?

**Comment by bmihaila** [comment]
Yes agree with the separation. I will have a second look at it the next days to see if something is missing.
